File: app/src/main/java/com/example/taxconnect/model/RequestItem.java
package com.example.taxconnect.model;

public class RequestItem {
    public static final int TYPE_CONVERSATION = 0;
    public static final int TYPE_BOOKING = 1;

    private int type;
    private Object data;

    public RequestItem(int type, Object data) {
        this.type = type;
        this.data = data;
    }

    public int getType() { return type; }
    public Object getData() { return data; }
}

File: tax/app/src/main/java/com/example/taxconnect/data/models/RequestItem.kt
package com.example.taxconnect.data.models

class RequestItem(
    val type: Int,
    val data: Any
) {
    var userModel: UserModel? = null
    var isReturning: Boolean = false

    companion object {
        const val TYPE_CONVERSATION = 0
        const val TYPE_BOOKING = 1
    }
}
